--- Logging error ---
Traceback (most recent call last):
  File "/home/wangzhaowei/anaconda3/envs/rongyf/lib/python3.6/logging/__init__.py", line 992, in emit
    msg = self.format(record)
  File "/home/wangzhaowei/anaconda3/envs/rongyf/lib/python3.6/logging/__init__.py", line 838, in format
    return fmt.format(record)
  File "/home/wangzhaowei/anaconda3/envs/rongyf/lib/python3.6/logging/__init__.py", line 575, in format
    record.message = record.getMessage()
  File "/home/wangzhaowei/anaconda3/envs/rongyf/lib/python3.6/logging/__init__.py", line 338, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "main.py", line 320, in <module>
    train_cnn_model('pnasnet5large_class3', gpu_ids, batch_size)
  File "main.py", line 276, in train_cnn_model
    logging.debug("Let's use ", len(gpu_ids), " GPUs!")
Message: "Let's use "
Arguments: (8, ' GPUs!')
Epoch     5: reducing learning rate of group 0 to 1.0000e-06.
Epoch    10: reducing learning rate of group 0 to 1.0000e-07.
Epoch    13: reducing learning rate of group 0 to 1.0000e-08.
